leapp/libraries/stdlib/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+:py:mod:`leapp.libraries.stdlib` represents a location for functions that otherwise would be defined multiple times across leapp actors
+ and at the same time are really useful for other actors.
+"""
+import six
+import subprocess
+import os
+
+
+def check_output(args, split=True):
+    """
+    Calls an external program and captures its ouput. Additionally the output is automatically utf-8 decoded,
```

Comment:
_Calls_ -> _Call_
Use imperative mood/command style, based on [PEP-257](https://www.python.org/dev/peps/pep-0257/) referred by PEP-8:
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

That reminds me to add PEP-257 to the contribution guidelines.

Reply:
updated